### PR TITLE
Added new error page

### PR DESCRIPTION
**Pull request by mickaelandrieu**
REFS https://github.com/backbee/BbCoreJs/issues/801, this is the new error page.

404_works

* manage 404, 500, custom errors,
* if connected, display the toolbar,
* suggest to come back to homepage

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
I need this one to be merged and deployed before => https://github.com/backbee/BackBee/pull/430

File: Resources/layouts/error/error.twig [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+</head>
+<body style="margin:0;padding:0;">
+    {% if this.isBBUser() %}
+        <link href="{{ this.getResourceUrl('toolbar/dist/css/bb-ui' ~ isMin ~ '.css') }}" rel="stylesheet">
+        <link href="{{ this.getResourceUrl('toolbar/dist/css/vendor' ~ isMin ~ '.css') }}" rel="stylesheet">
+        {{ this.bbtoolbar()|raw }}
+    {% else %}
+        <link href="{{ this.getResourceUrl('toolbar/dist/css/bb-ui-login' ~ isMin ~ '.css') }}" rel="stylesheet">
+    {% endif %}
```
Comment (ndufreche) [inline Resources/layouts/error/error.twig]: You have to only use bbtoolbar helper and correct the helper to integrate the stylesheet every time.

File: Resources/layouts/error/js/error_edgeActions.js [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+         $(document).ready(function(){
+             scaleStage();
+         });
+
+      });
+      //Edge binding end
+
+      Symbol.bindElementAction(compId, symbolName, "${bt_back}", "click", function(sym, e) {
+         // insérer le code du clic de souris ici
+         window.open(window.location.origin?window.location.origin+'/':window.location.protocol+'/'+window.location.host+'/',"_self");
```
Comment (ndufreche) [inline Resources/layouts/error/js/error_edgeActions.js]: For the link to home, create a href not visible, use the backbee helper to get the good base url.

**Comment by eric-chau** [comment]
Good job, :+1: thank you @mickaelandrieu.

**Pull request merged**